meetings/cpansec-minutes-2025-01-22.md

```diff
@@ -0,0 +1,124 @@
+---
+layout: page
+toc: true
+meeting_time: 2025-01-22 16:00 UTC
+title: Minutes 2025-01-22
+---
+
+## Agenda & Meeting Details
+
+* 2025-01-22 16:00 UTC on #cpansec-discussion on Matrix
+
+## 15:30 UTC – Pre-meeting socializing
+
+*   Socializing & getting up to speed before the meeting starts properly
+*   Discuss organizing projects, swimlanes and issues (...)
+
+## 16:05 UTC – Meeting start
+
+### Welcome
+
+*   Meeting chair: 
+*   Meeting scribe: 
+
+### Attendees, absents & regrets
+
+*   Attendees
+    * @sjn, 
+*   Regrets
+    * @thibaultduponchelle
+
+### Approve previous meeting minutes
+*   Previous meeting minutes was approved by XXX and merged by XXX
+
+
+### Quick summary of current work (Grouped by project)
+, 
+1.  [CPAN Metadata & Software Bills of Materials](https://github.com/orgs/CPAN-Security/projects/1)
+    *   …
+2.  [CPAN Privacy and Compliance](https://github.com/orgs/CPAN-Security/projects/9)
+    *   …
+3.  [CPAN Provenance & Supply Chain Security](https://github.com/orgs/CPAN-Security/projects/3)
+    *   …
+4.  [CPAN Security Outreach & Information](https://github.com/orgs/CPAN-Security/projects/12)
+    *   …
+5.  [CPAN Security Patch Tooling](https://github.com/orgs/CPAN-Security/projects/11)
+    *   …
+6.  [CPAN Software Composition Analysis](https://github.com/orgs/CPAN-Security/projects/6)
+    *   …
+7.  [CPAN Transparency Logs](https://github.com/orgs/CPAN-Security/projects/2)
+    *   …
+8.  [CPAN Vulnerability Index](https://github.com/orgs/CPAN-Security/projects/10)
+    *   …
+9.  [CPANSec Governance, Policy & Funding](https://github.com/orgs/CPAN-Security/projects/7)
+    *   …
+10. [CPAN Secure by Default](https://github.com/orgs/CPAN-Security/projects/15)
+    *   …
+11. [CPAN Software Composition Analysis](https://github.com/orgs/CPAN-Security/projects/6)
+    *   …
+
+### TLS/HTTPS/CSPRNG/DSA in core
+- 
+
+### Funding drive
+- @sjn will work on that after FOSDEM - will need to know who is open to do (funded) work
+    - @stigtsp 1d/w
+
+### Ongoing vulnerabilities
+- 
+
+### Author guide to generating random values for security
+- 
```

Comment:
Probably mention issues with Crypt::URandom? @Leont has worked on Crypt::SysRandom, so maybe mention that. And @timlegge has taken over Crypt::Random so if issues are fixed with that, it might also be worth adding.

@stigtsp has suggested discussions about performance of reading /dev/urandom vs sys calls, so there is also more to add about that although I want to work on benchmarks with forking when I can.

Reply:
I don't think we specifically discussed these topics during the meeting.

Did we? cc: @stigtsp @timlegge

Reply:
I'll merge the minutes now; Please feel free to raise an issue if this really needs to be included!